tests/Integration/Behaviour/Features/Context/Domain/CategoryFeatureContext.php

```diff
@@ -67,9 +67,6 @@ class CategoryFeatureContext extends AbstractDomainFeatureContext
         'down' => 1,
     ];
 
-    /** @var int */
-    private $defaultLanguageId;
-
     /** @var string */
     private $psCatImgDir;
```

Comment:
@jolelievre unused

Reply:
Thanks I forgot to remove it, that was the initial point :D